readme.md

```diff
@@ -310,6 +310,25 @@ If you are simply adding additional audits/gatherers or tweaking flags, you can
 }
 ```
 
+### helpText guidelines
+
+Keep the `helpText` of an audit as short as possible. When a reference doc for the audit exists on
+developers.google.com/web, the `helpText` should only explain *why* the user should care
+about the audit, not *how* to fix it.
+
+Do:
+
+    Serve images that are smaller than the user's viewport to save cellular data and
+    improve load time.
```

Comment:
Maybe add a markdown link in the "Do" example so it's clear you should do this when you have a link :)